.github/workflows/run_periodic_tests.yml

```
@@ -36,8 +36,7 @@ jobs:
       - name: Check style
         run: |
           python -m pip install pre-commit
-          git add .
-          pre-commit run ruff
+          pre-commit run --all
```

```suggestion
          pre-commit run --all-files
```

(Surprisingly) just `--all` works too, but imo it would be great to match the official CLI reference

---

Hmm I wonder if that was a change since I was doing it. I have always done ```--all```
```suggestion
          pre-commit run -a
```